assets/js/googlesitekit-base.js

```diff
@@ -21,7 +21,7 @@
 /**
  * Internal dependencies
  */
-import { clearWebStorage } from './util/standalone';
+import { clearCache } from './googlesitekit/api/cache';
```

Comment:
This issue has #6250 as a dependency to avoid this, as we do not want to include this here. This whole file is going away so this change is only a problem if this were released before it's dependent issue.

Reply:
Understood and completely agreed. I'll keep this issue in Execution for now until the dependent is merged, after which I'll go ahead and update the branch to be up to date with `develop`. That way, I don't have to make additional changes, like bringing back the already removed `clearWebStorage`.

Reply:
This is back in CR for another go, thanks!